securedrop/i18n_tool.py

```diff
@@ -37,6 +37,7 @@ class I18NTool(object):
     SUPPORTED_LANGUAGES = {
         'ar': {'name': 'Arabic', 'desktop': 'ar', },
         'ca': {'name': 'Catalan', 'desktop': 'ca', },
+        'cs': {'name': 'Czech', 'desktop': 'cs', },
```

Comment:
let's run `securedrop/bin/dev-shell ./i18n_tool.py --verbose update-docs` now so that we don't forget to update the docs before the release (only relevant for adding new languages)

Reply:
Done.